framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Deployment.java

```diff
@@ -17,10 +17,39 @@
 
 import java.util.List;
 
+/**
+ * Cloud deployment representation. Description of deployed aplication.
+ */
 public interface Deployment {
+
+    /**
+     * Return cloud project name.
+     *
+     * @return project name
+     */
     String getNamespace();
+
+    /**
+     * Change number of pods available for the deployment.
+     *
+     * @param instances Number of pods to be available.
+     */
     void scale(int instances);
+
+    /**
+     * Wait until Deployment is ready to use. This method wait until all
+     * instances of deployment are initialized and ready and for router to
+     * expose url.
+     */
     void waitForScale();
+
+    /**
+     * Return list of all instances available for the deployment.
```

Comment:
Instances available for deployment? Aren't those already running instances of the deployment? But maybe I don't understand what an instance of deployment is..